contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/check_gpio.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+import argparse
+import os
+from contextlib import contextmanager
+
+# from checkbox_support.snap_utils.snapd import Snapd
+from checkbox_support.snap_utils.snapd import Snapd
+from checkbox_support.snap_utils.system import get_gadget_snap
+import requests
+
+
+def list_gpio_slots(snapd, gadget_name):
```

Comment:
```suggestion
def list_gpio_slots(snapd: object, gadget_name: str) -> Dict[str, Dict[str, int]]:
```

Reply:
If we are using `Dict`, I think we may need the `from typing import Dict`?